### ISSUE TITLE
[Bug]: Select an existing column to add always includes "database-plugin" key, using folder db in empty folder.

### ISSUE THREAD

**Issue by rhhub** [RafaelGB/obsidian-db-folder#843]
### Steps to reproduce
1. Create new folder in obsidian.
2. Right-click folder, select New-db folder.
3. Open Untitled database in respective folder.
4. Click + to add column to table.
5. Under "Select an existing column to add", put cursor in Search column text field.
6. Note "database-plugin" shows up.

I expected it to not show up.

### Thoughts
I suspect this comes from the Untitled database file created in this instance.

I think any modification should only omit the entry of the current database. If there are more databases then those should still lead to "database-plugin showing up.

### What platform were you using?

Desktop

### Version of the plugin

3.3.2

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
I think we do not need to exclude this default field.

You can apply formulas for its value and could be use cases where is needed but I am open to discussion

**Comment by rhhub** [comment]
This caused a little bit of friction because I was using the key suggestion to find keys to clean up. I found it odd that it would suggest a column with the database-plugin key even though the table wouldn't present a file containing it. When I went to sort the column by ascending or descending, no file with the database-plugin key would present itself. To clarify, I'm suggesting the ignoring of the database-plugin entry in the one file that is presenting the table.
